.env

```diff
@@ -1,3 +1,6 @@
+DJANGO_BASE_URL=https://dtp-stat.ru
+DJANGO_CONTENT_FALLBACK=false
```

Comment:
It might be best to keep this feature off during local dev by default. It helps detect what assets are missing.